tests/Suite/Component/UtilsTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
there's a file named functions.php which holds a couple of general purpose helper functions. Perhaps that would be a nice place to put `isAssociativeArray`

Reply:
updating now